core/embed/rust/src/ui/layout_eckhart/ui_firmware.rs

```diff
@@ -259,11 +323,30 @@ impl FirmwareUI for UIEckhart {
     }
 
     fn confirm_properties(
-        _title: TString<'static>,
-        _items: Obj,
-        _hold: bool,
+        title: TString<'static>,
+        items: Obj,
+        hold: bool,
     ) -> Result<impl LayoutMaybeTrace, Error> {
-        Err::<RootComponent<Empty, ModelUI>, Error>(Error::ValueError(c"not implemented"))
+        let paragraphs = PropsList::new(
+            items,
+            &theme::TEXT_SMALL_LIGHT,
+            &theme::TEXT_MEDIUM,
```

Comment:
```suggestion
            &theme::TEXT_MONO_LIGHT,
```

Let's do the same `TextStyle` also for values. It's the same in older layouts as well. We probably only need 2 textstyles but that is outside the scope of this PR.

Reply:
https://github.com/trezor/trezor-firmware/pull/4899/commits/ece82cddc7b03fb0c57a0609e605ce3d737cf1e2